Cake.Recipe/Content/toolsettings.cake

```diff
@@ -34,7 +34,7 @@ public static class ToolSettings
     public static string FixieTool { get; private set; }
 
     public static void SetToolPreprocessorDirectives( 
-        string codecovTool = "#tool nuget:?package=codecov&version=1.4.0",
+        string codecovTool = "#tool nuget:?package=codecov&version=1.7.2",
```

Comment:
This was a surprise, did not expect the addin discoverer to pick up the new location of the tool references.

Reply:
I had a quick chat with @jericho and he said the change we made shouldn’t affect things. 😄

Reply:
@admiringworm any objections to pulling this in?

Reply:
@admiringworm any objections to pulling this in?

Reply:
none, it shouldn't break anything, users just won't get the full benefit until I can get around to making some changes to the logic in Cake.Recipe.

Reply:
Addin.Discoverer scans all `.cake` files in the `Cake.Recipe/Content` folder. So, as long as the file remain in this location the discoverer will have no problem discovering addin references in the files.